7.application_configuration.md

```diff
@@ -70,70 +68,61 @@ The specification defines three major things: Parameters that can be overridden
-| `scopes` | [`[]Scope`](#scope) | N | | Application scope definitions. If defined, scopes are accessible by components from any other application configurations. |
 | `components` | [`[]Component`](#component) | N | | Component instance definitions. |
 
-
-### Variable
-The Variables section defines variables that may be used elsewhere in the application configuration. The `variable` section provides a way for an application operator to specify common values that can be substituted into multiple other locations in this configuration (using the `[fromVariable(VARNAME)]` syntax).
-
-| Attribute | Type | Required | Default Value | Description |
-|-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The parameter's name. Must be unique per configuration. |
-| `value` | `string` | Y | | The scalar value. |
-
-A `name` is any Unicode letter or numeric character, `_`, `-`, and `.`. A `value` is any sequence of printable Unicode characters.
-
-The function `[fromVariable(VARNAME)]` (where `VARNAME` corresponds to the `name` field in a variable definition) may be used in predefined locations (parameter values and properties objects) to access the `value` of a parameter.
-
-### Scope
-The scope section defines application scopes that will be created with this application configuration.
-
-| Attribute | Type | Required | Default Value | Description |
-|-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The name of the application scope. Must be unique to the deployment environment. |
-| `type` | `string` | Y | | The fully-qualified GROUP/VERSION.KIND name of the application scope. |
-| `properties` | [`Properties`](#properties) | N | | Overrides of properties that are exposed by this scope. |
-
-> The name field must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
-
 ### Component
 This section defines the instances of components to create with this application configuration.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
 | `componentName` | `string` | Y | | The name of the component to create an instance of. |
-| `instanceName` | `string` | Y | | The name of the instance of this component. |
 | `parameterValues` | [`[]ParameterValue`](#parameterValue) | N | | Overrides of parameters that are exposed by the application scope type defined in `type`. |
 | `traits` | [`[]Trait`](#trait) | N | | Specifies the traits to attach to this component instance. |
-| `applicationScopes` | `[]string` | N | | Specifies the application scopes that the component should be deployed into. Each item is a name referenced to the pre-defined Scopes. |
+| `scopes` | [`[]Scope`](#scope) | N | | Specifies the application scopes that the component should be deployed into. |
```

Comment:
why not call it application scopes anymore?

Reply:
1. scopes is more short and also can be understand.
2. scope is a group of components. it's more like componentScope not applicationScope.

Reply:
Agree. Let's call it "scopes".